Example/Podfile

```diff
@@ -10,7 +10,7 @@ target 'AccessibilitySnapshotDemo' do
 	target 'SnapshotTests' do
 		inherit! :search_paths
 
-		pod 'AccessibilitySnapshot', :path => '../AccessibilitySnapshot.podspec'
+		pod 'AccessibilitySnapshot/iOSSnapshotTestCase', :path => '../AccessibilitySnapshot.podspec'
 		pod 'AccessibilitySnapshot/SnapshotTesting', :path => '../AccessibilitySnapshot.podspec'
```

Comment:
Technically we don't need the `/SnapshotTesting` part here but equally might be worth keeping to be explicit about which specs we are using?

Reply:
Yeah, I think it's probably good to be explicit about it here. One less change to make if/when we add a new subspec or change the default subspecs.